lib/src/web_socket/w_socket_subscription.dart

```diff
@@ -58,8 +58,8 @@ class WSocketSubscription<T> implements StreamSubscription<T> {
   }
 
   @override
-  Future/*<E>*/ asFuture/*<E>*/([var/*=E*/ futureValue]) {
-    final c = new Completer/*<E>*/();
+  Future<E> asFuture<E>([E futureValue]) {
```

Comment:
😃

Reply:
I suppose this isn't exactly a breaking change .. but does make the return type more specific.

Reply:
Actually since it’s a method-generic, I think `Future<E>` ends up being the same as `Future` for existing consumptions that don’t leverage the generic because both equate to `Future<dynamic>`, right?

Reply:
That is correct. Sorry, I wasn't asking for any change here .. just noticing.